erpnext/accounts/utils.py

```diff
@@ -740,3 +740,32 @@ def create_payment_gateway_account(gateway):
 	except frappe.DuplicateEntryError:
 		# already exists, due to a reinstall?
 		pass
+
+
+def get_party_shipping_address(doctype, name):
```

Comment:
Keep this function in party.py itself. accounts/utils.py is specific to accounting functions